src/controllers/simplified-account/settings/stripe-details/director/director.controller.js

```diff
@@ -18,10 +18,11 @@ async function post (req, res, next) {
     await validation.run(req)
   }
   const errors = validationResult(req)
-    .formatWith(({ msg, path }) => {
+    .formatWith(({ msg, path, type }) => {
```

Comment:
What does adding a `type` do?

Reply:
see https://express-validator.github.io/docs/migration-v6-to-v7/#telling-error-types-apart